english/blah-balh/blah-blah.md

```diff
@@ -2,6 +2,6 @@ Say something, just talk, so everybody can listen to the voice you have to offer
 
 Be as smart as you can be in your voice so you don't sound stupid.
 
-And sing if possible. Singing cheers people up.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.